test/tools/yulInterpreter/EwasmBuiltinInterpreter.h

```diff
@@ -109,14 +113,18 @@ class EwasmBuiltinInterpreter
 	/// Does not adjust msize, use @a accessMemory for that
 	void writeMemoryByte(uint64_t _offset, uint8_t _value);
 
-	/// Helper for eth.* builtins. Writes to memory (big-endian) and always returns zero.
+	/// Helper for eth.* builtins. Writes to memory (little-endian) and always returns zero.
 	void writeU256(uint64_t _offset, u256 _value, size_t _croppedTo = 32);
 	void writeU128(uint64_t _offset, u256 _value) { writeU256(_offset, std::move(_value), 16); }
-	void writeAddress(uint64_t _offset, u256 _value) { writeU256(_offset, std::move(_value), 20); }
-	/// Helper for eth.* builtins. Reads from memory (big-endian) and returns the value;
+	/// Helper for eth.* builtins. Writes to memory (as a byte string).
+	void writeBytes32(uint64_t _offset, util::h256 _value) { accessMemory(_offset, 32); writeMemory(_offset, _value.asBytes()); }
```

Comment:
Why have both `writeBytes32` and `writeU256`?

Reply:
Different byte ordering.

Reply:
Could that be made more explicit in the function's names? Maybe in the little-endian case only

Reply:
These names follow the "ewasm spec".